### PR TITLE
fix allignment

### PR DESCRIPTION
**Pull request by leleOFA**
## Short description
fix allignment  button with title of page when has in mobile phone
## List of changes proposed in this pull request
insert condition with mobile hook in margin x for button

### CODE REVIEW

File: packages/pn-personagiuridica-webapp/src/components/Notifications/GroupSelector.tsx [hunk 1]
```diff
@@ -33,7 +35,11 @@ const GroupSelector: React.FC<Props> = ({ currentGroup, onGroupSelection }) => {
   }, []);
 
   return (
-    <Box data-testid="groupSelector" display="inline-block" sx={{ verticalAlign: 'middle', mx: 2 }}>
+    <Box
+      data-testid="groupSelector"
+      display="inline-block"
+      sx={{ verticalAlign: 'middle', mx: isMobile ? 0 : 2 }}
+    >
       <Box>
```
Comment (AndreaCimini90) [inline packages/pn-personagiuridica-webapp/src/components/Notifications/GroupSelector.tsx:43]: Remove extra box

File: packages/pn-personagiuridica-webapp/src/components/Notifications/GroupSelector.tsx [hunk 2]
```diff
@@ -33,7 +35,11 @@ const GroupSelector: React.FC<Props> = ({ currentGroup, onGroupSelection }) => {
   }, []);
 
   return (
-    <Box data-testid="groupSelector" display="inline-block" sx={{ verticalAlign: 'middle', mx: 2 }}>
+    <Box
+      data-testid="groupSelector"
+      display="inline-block"
```
Comment (AndreaCimini90) [inline packages/pn-personagiuridica-webapp/src/components/Notifications/GroupSelector.tsx:40]: change this property with `display={{ xs: 'block', lg: 'inline-block' }}`

File: packages/pn-personagiuridica-webapp/src/components/Notifications/GroupSelector.tsx [hunk 3]
```diff
@@ -33,7 +35,11 @@ const GroupSelector: React.FC<Props> = ({ currentGroup, onGroupSelection }) => {
...
+      sx={{ verticalAlign: 'middle', mx: isMobile ? 0 : 2 }}
```
Comment (AndreaCimini90) [inline packages/pn-personagiuridica-webapp/src/components/Notifications/GroupSelector.tsx:41]: change this property with `sx={{ verticalAlign: 'middle', mx: { xs: 0, lg: 2 }, mt: { xs: 1, lg: 0 } }}`

**Review by AndreaCimini90 — CHANGES_REQUESTED** [review]

File: packages/pn-personagiuridica-webapp/src/components/Notifications/GroupSelector.tsx (hunk 1 above)
Comment (leleOFA) [reply packages/pn-personagiuridica-webapp/src/components/Notifications/GroupSelector.tsx]: done

File: packages/pn-personagiuridica-webapp/src/components/Notifications/GroupSelector.tsx (hunk 2 above)
Comment (leleOFA) [reply packages/pn-personagiuridica-webapp/src/components/Notifications/GroupSelector.tsx]: done

File: packages/pn-personagiuridica-webapp/src/components/Notifications/GroupSelector.tsx (hunk 3 above)
Comment (leleOFA) [reply packages/pn-personagiuridica-webapp/src/components/Notifications/GroupSelector.tsx]: done

**Review by AndreaCimini90 — APPROVED** [review]

**Pull request merged**